runner/app/routes/llm.py

```diff
@@ -0,0 +1,110 @@
+import logging
+import os
+from typing import Annotated
+from fastapi import APIRouter, Depends, Form, status
+from fastapi.responses import JSONResponse, StreamingResponse
+from fastapi.security import HTTPAuthorizationCredentials, HTTPBearer
+from app.dependencies import get_pipeline
+from app.pipelines.base import Pipeline
+from app.routes.util import HTTPError, LLMResponse, http_error
+import json
+
+router = APIRouter()
+
+logger = logging.getLogger(__name__)
+
+RESPONSES = {
+    status.HTTP_200_OK: {"model": LLMResponse},
+    status.HTTP_400_BAD_REQUEST: {"model": HTTPError},
+    status.HTTP_401_UNAUTHORIZED: {"model": HTTPError},
+    status.HTTP_500_INTERNAL_SERVER_ERROR: {"model": HTTPError},
+}
+
+
+@router.post("/llm",
+             response_model=LLMResponse, responses=RESPONSES, operation_id="genLLM",)
+@router.post("/llm/", response_model=LLMResponse, responses=RESPONSES, include_in_schema=False)
```

Comment:
Can you add metadata for OpenAPI docs&SDK? Here's a suggestion to make it easy:
```suggestion
@router.post(
    "/llm",
    response_model=LLMResponse,
    responses=RESPONSES,
    operation_id="llm",
    description="Generate text using a language model.",
    summary="LLM",
    tags=["generate"],
    openapi_extra={"x-speakeasy-name-override": "llm"},
)
@router.post(
    "/llm/",
    response_model=LLMResponse,
    responses=RESPONSES,
    include_in_schema=False,
)
```

I changed the `operation_id` to just `llm` since Rick mentioned you wanted not to have the `gen` prefix here. As I mentioned in Discord, this `operation_id` doesn't matter for the SDK so feel free to skip it in your API if you think it's more important than the consistency.